ckanext/budgetviz/groupbarchart_munc/theme/public/custom.css

```diff
@@ -15,6 +15,9 @@
 }
 
 .chartarea {
-    margin-top: 20px;
+    padding: 10px 25px;
```

Comment:
I meant desktop resolutions only. Not the Mobile or tab ones. 

There  are 13 inch laptops also. Would it work on them?